backend/mcp_backend/routers/mcp.py

```
@@ -0,0 +1,963 @@
+    api_configs = get_config_value(request, "MCP_API_CONFIGS", {})
+    filtered_configs = {key: value for key, value in api_configs.items() if key in keys}
+    set_config_value(request, "MCP_API_CONFIGS", filtered_configs)
+
+    return {
+        "ENABLE_MCP_API": get_config_value(request, "ENABLE_MCP_API", False),
+        "MCP_BASE_URLS": get_config_value(request, "MCP_BASE_URLS", []),
+        "MCP_API_CONFIGS": get_config_value(request, "MCP_API_CONFIGS", {}),
+    }
+
+
+async def get_tools_from_mcp_server(url: str, idx: int, api_config: Dict):
```

Ruff (0.12.2)

Unused function argument: `idx`

(ARG001)